plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -585,12 +585,24 @@ function MosaicMenuItem:update()
                 cover_bb_used = true
                 -- Let ImageWidget do the scaling and give us a bb that fit
                 local _, _, scale_factor = BookInfoManager.getCachedCoverSize(bookinfo.cover_w, bookinfo.cover_h, max_img_w, max_img_h)
+                local consistent_aspect_ratio_covers = BookInfoManager:getSetting("consistent_aspect_ratio_covers") and true or false
+
+                local scale_factor_img = consistent_aspect_ratio_covers and nil or scale_factor
+                if consistent_aspect_ratio_covers then scale_factor_img = nil end
```

Comment:
I think you got there by _trial and error and add kludge to fix the error_ :)
I guess what you want is just:
`local scale_factor_img = not consistent_aspect_ratio_covers and scale_factor or nil`